projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/credentials/AWSServiceAccountCredentialsService.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2023-2023 VMware, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.vmware.taurus.service.credentials;
+
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.auth.BasicSessionCredentials;
+import com.amazonaws.auth.STSAssumeRoleSessionCredentialsProvider;
+import com.amazonaws.services.securitytoken.AWSSecurityTokenService;
+import com.amazonaws.services.securitytoken.AWSSecurityTokenServiceClientBuilder;
+import com.amazonaws.services.securitytoken.model.AssumeRoleRequest;
+import java.util.UUID;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AWSServiceAccountCredentialsService {
+
+  @Value("${datajobs.aws.serviceAccountSecretAccessKey:}")
```

Comment:
I really recommend against using @Value on fields. It is better to use constructor injection with@Value. 

Using constructor injection allows for mandatory dependencies to be passed in at the time of object creation, making it clear which dependencies are required for the object to function properly.
 Very importantly it also makes it easier to write unit tests, as dependencies can be easily mocked and injected.